notebooks/human_keypoint_pose.ipynb

```diff
@@ -0,0 +1,247 @@
+    "    COCO_KEYPOINTS,\n",
+    "    COCO_SKELETON,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4bc528fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "json_file = \"/home/david.wang/datasetinsights/tests/mock_data/coco/annotations/keypoints.json\"\n",
+    "f = open(json_file)\n",
+    "data = json.load(f)\n",
+    "annotations = data[\"annotations\"]\n",
+    "keypoints_list = []\n",
+    "for k in annotations:\n",
+    "    keypoints_list.append(k[\"keypoints\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "260640fd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "from matplotlib import collections as mc\n",
+    "from matplotlib import cm, colors\n",
+    "from seaborn.distributions import _DistributionPlotter as db\n",
+    "\n",
+    "sns.set_theme(style=\"ticks\", font_scale=1.25, color_codes=True,\n",
+    "palette='colorblind', font='sans-serif', context='paper', rc={\"lines.linewidth\": 1.5})\n",
+    "\n",
+    "def plot_heatmaps(kp_dict, cmap=\"Reds\", color=\"red\"):\n",
+    "    keypoints_name = COCO_KEYPOINTS\n",
+    "\n",
+    "    for key in keypoints_name:\n",
+    "        fig, ax = plt.subplots(dpi=100, figsize=(8,8))\n",
+    "        \n",
+    "        \n",
+    "        sns.kdeplot(x=kp_dict[key][\"x\"], y=kp_dict[key][\"y\"], \n",
+    "                    shade=True, label=key, ax=ax, alpha=0.85, color=color, \n",
+    "                    cbar = False,)\n",
+    "\n",
+    "        ax.set_xlim(-2.5, 2.5)\n",
+    "        ax.set_ylim(-2.5, 2.5)\n",
+    "        \n",
+    "        ax.spines[\"left\"].set_position(\"center\")\n",
+    "        ax.spines[\"bottom\"].set_position(\"center\")\n",
+    "\n",
+    "        # Eliminate upper and right axes\n",
+    "        ax.spines[\"right\"].set_color(\"none\")\n",
+    "        ax.spines[\"top\"].set_color(\"none\")\n",
+    "\n",
+    "        # # Turn off tick labels\n",
+    "        ax.set_yticklabels([])\n",
+    "        ax.set_xticklabels([])\n",
+    "        \n",
+    "        props = dict(boxstyle='square', facecolor='wheat', alpha=0.1, edgecolor='black')\n",
+    "        t = key.split(\"_\")\n",
+    "        t = [x.capitalize() for x in t]\n",
+    "        t = \" \".join(t)\n",
+    "        textstr = t\n",
+    "        ax.text(0.005, 0.080, textstr, transform=ax.transAxes, fontsize=36, va='top' )\n",
+    "        \n",
+    "        ax.patch.set_edgecolor('black')\n",
+    "\n",
+    "        ax.patch.set_linewidth('1')\n",
+    "\n",
+    "        # Invert axes\n",
+    "        ax.invert_yaxis()\n",
+    "        \n",
+    "        ax.set_aspect('equal')\n",
+    "        \n",
+    "        plt.show()\n",
+    "#         plt.savefig(f\"{fig_path}{key}_heatmap.png\", bbox_inches='tight', pad_inches=0.15)\n",
```

Comment:
Why is this commented?

Reply:
It's intended to just show plots in the notebook example, without saving.